admin-dev/themes/default/scss/modules/_variables.scss

```diff
@@ -169,14 +169,18 @@ $paddingLeftMobile: 0.625rem;
 $widthSidebarSubmenu: 200px;
 $menu-item-size: 34px;
 $min-height: 950px;
-$gray-dark-text: #bebebe;
-$gray-dark-text-hover: #fff;
+$gray-dark-link: #fff;
+$gray-dark-link-hover: #fff;
```

Comment:
Normally I would say use the $white variable, but if we wan'ts to change the $white variable I think we don't want menu links to be #000 but a grayer color so it's fine to me!